### PR TITLE
4.50.0 changelog

### PR DESCRIPTION
**Pull request by pmaytak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
+4.50.0
```
Comment (gladjohn) [inline CHANGELOG.md:1]: - do we need to add better exception handling in this scenarios? [#3883](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3883) and [#3614](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3614)
- [Bug] Logs are created even if logging is disabled [#3901](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3901) - did not see info on this
- [Bug] Managed Identity - Update logs or docs to clarify MI correlation ID is different than MSALs [#3908](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3908)
- [Request] WAM support for no scopes [#3675](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3675)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+- WAM preview now honors the login hint. See [3301](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3301).  
+- WAM preview now allows to sign in with an account different from the provided login hint. See [3929](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3929).  
+- Fixed an `ApiContractViolation` exception in WAM preview when signing out. See [3685](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3685).  
+- MSAL now allows passing no scopes when using WAM preview. See [3675](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3675).  
+- Added a more descriptive error message when combined flat user and app cache is used. Use a partitioned token cache (for ex. distributed cache like Redis) or separate files for app and user token caches. See [3218](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3218).  
+- When broker is enabled, MSAL will now use the refresh token from the broker instead of a locally cached one. See [3613](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3613).
```
Comment (gladjohn) [inline CHANGELOG.md:17]: Is this just for the new windows broker?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+### New Features
+- Extended managed identity experimental functionality with support for Azure Cloud Shell. See [3832](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3832).  
+- Added support for PII logging for WAM preview. See [3845](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3845), [3822](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3822).  
+
+
+### Bug Fixes
+- Fixed JSON serialization issues for apps running on .NET 7. See [3892](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3892).  
+- Improved logging performance. See [3901](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3901).  
+- Fixed `Unable to load DLL 'msalruntime'` exception for apps that use WAM preview and are packaged as MSIX. See [3740](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3740).  
+- WAM preview now honors the login hint. See [3301](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3301).
```
Comment (gladjohn) [inline CHANGELOG.md:12]: maybe a link to WAM Preview docs? would be nice for anyone new seeing this for the first time to see the WAM docs at https://aka.ms/msal-net-wam

**Review by gladjohn — APPROVED** [review]

**Review by neha-bhargava — APPROVED** [review]

**Review by bgavrilMS — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (pmaytak) [reply CHANGELOG.md:17]: Both.

**Pull request merged**